### ISSUE TITLE
Duplicate and contradictory CSS rule in viewer.css

### ISSUE THREAD

**Issue by stephanrauh** [mozilla/pdf.js#11264]
Hi team,

I've just noticed a duplicate definition of a CSS rule:

```CSS
.toolbarField {
  ...
  border: 1px solid transparent;
   ...
  border: 1px solid hsla(0,0%,0%,.35);
  border-color: hsla(0,0%,0%,.32) hsla(0,0%,0%,.38) hsla(0,0%,0%,.42);
```

IMHO there's one or two rules too many.

For the sake of convenience, here's the [link to the line in the repo](https://github.com/mozilla/pdf.js/blob/d7f651aa9885c1848d1c8694c038bf4e0e09d6fe/web/viewer.css#L1030).

Best regards,
Stephan

**Comment by Virtual1** [comment]
I'd like to claim this, if possible? See if I can figure out & fix it. 
Thanks 
Jessica

**Comment by timvandermeij** [comment]
It's yours! I would check which rules are actually applied by the browser for `.toolbarField` using the developer console. I'm pretty sure the `border: 1px solud transparent` is overwritten by the other `border` entry, but I don't know which border color is used. I also think there are a few more CSS entries where the border has a color, but there is also a separate `border-color` property; I think the latter overrules the former.

**Comment by Zdobnov** [comment]
As this issue was still opened, I had an initiative: https://github.com/mozilla/pdf.js/pull/11268. This PR refactors the `viewer.css` a bit + removes duplications mentioned above.
I hope it will be useful.
Thanks.

**Comment by Virtual1** [comment]
Hi, I guess @Zdobnov fixed it, let me know if there's still something to check here?
Thanks,
Jessica

**Comment by Snuffleupagus** [comment]
Given that someone had already expressed interest in working on this *less than a day ago*, in https://github.com/mozilla/pdf.js/issues/11264#issuecomment-544543001 above, generally speaking it seems like somewhat bad form to just "steal" the issue.

**Comment by stephanrauh** [comment]
@Virtual1 If you want to help the project, you could have a quick look at #11264 and confirm this bug has been solved. You could also have a look at a second code duplication. Originally, I wanted to open a second ticket, but chances are it has been solved, so we can probably simplify the process:

![image](https://user-images.githubusercontent.com/3045767/67318608-00012b00-f50c-11e9-8f35-798b6a82ca83.png)
https://sonarcloud.io/project/issues?id=stephanrauh_ngx-extended-pdf-viewer&open=AW3qm3-l-dJmagWAuDEh&resolved=false&types=BUG

BTW: as soon as someone has confirmed the two duplicate rules have been fixed, we can close this ticket. :)

**Comment by timvandermeij** [comment]
Closing since both problems are fixed by #11268. There was a lot of interest in the issue, so thank you all for helping out. If there are more things to fix in the CSS, please open a new issue.
